src/api-addr.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const Key = require('interface-datastore').Key
+
+const apiFile = new Key('api')
+
+module.exports = (store) => {
+  return {
+    /**
+     * Get the current configuration from the repo.
+     *
+     * @param {function(Error, Object)} callback
+     * @returns {void}
+     */
+    get (callback) {
+      store.get(apiFile, (err, value) => {
+        if (err) {
+          return callback(err)
+        }
+
+        callback(err, value && value.toString())
+      })
+    },
```

Comment:
since you are doing the check for value, this method can be simplified to:

```
get (callback) {
  store.get(apiFile, (err, value) => callback(err, value && value.toString()))
}
```